elementary/monitor/cli.py

```diff
@@ -291,13 +300,16 @@ def send_report(
     anonymous_tracking.track_cli_start('monitor-send-report', get_cli_properties(), ctx.command.name)
     try:
         config.validate_send_report()
+        # If file name contains a folders (is a path),
+        # we want to make sure that we create the report localy without the folders of the full path.
```

Comment:
Update comment to something like - "bucket-file-path determines the path of the report in the bucket. If this path contains folders we extract the report file name to first save the report locally"